test/integration/sigint.spec.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var helpers = require('./helpers');
+var invokeMocha = helpers.invokeMocha;
+var invokeSubMocha = helpers.invokeSubMocha;
+
+var fixturePath = require.resolve('./fixtures/sigint.fixture');
+
+describe('SIGINT handling', function() {
+  describe('using "mocha" wrapper', function() {
+    describe('when SIGINT received', function() {
+      it('should terminate itself w/ SIGTERM', function(done) {
```

Comment:
How/why did we switch from SIGINT to SIGTERM?
This by itself seems wrong...

Reply:
I don't actually remember why other than it was that `SIGINT` just wasn't cutting it.

FWIW, I haven't heard complaints about it since...